### ISSUE TITLE
False-positive on Angular aria-label binding

### ISSUE THREAD

**Issue by JeffreyNordlieTR** [dequelabs/axe-linter-action#53]
It appears to not be detecting that the button has an aria-label:
![Image](https://github.com/user-attachments/assets/d58cb5d4-c924-468d-8534-ea8c0b0db6f9)

**Comment by WilcoFiers** [comment]
@JeffreyNordlieTR thank you for raising this. Yeah that should definitely pass. Surprising.

**Comment by sergei-startsev** [comment]
@zlayaAvocado there're no linter overrides in our project. File renaming isn't an option, the existing convention is widely used across the project, moreover, I don't believe it'll help in any way.

**Comment by zlayaAvocado** [comment]
> [@zlayaAvocado](https://github.com/zlayaAvocado) there're no linter overrides in our project. File renaming isn't an option, the existing convention is widely used across the project, moreover, I don't believe it'll help in any way.

Totally understand! Since linting is done based on file extensions, could you try setting up an override using an `axe-linter.yml` file so that your `.html` files are linted with the Angular linter instead? The override should look something like this:

```
overrides:
  - files: 
    - "**/*.html"
    linter: angular
```

**Comment by sergei-startsev** [comment]
The override seems to address the issue. Are there any extension mappings we should know about? Is this documented?

**Comment by sergei-startsev** [comment]
@zlayaAvocado I don't see that the doc you attached covers my question. Where can I find the predefined extension->linter mapping?

**Comment by zlayaAvocado** [comment]
> [@zlayaAvocado](https://github.com/zlayaAvocado) I don't see that the doc you attached covers my question. Where can I find the predefined extension->linter mapping?

Oh, I see what you mean. Here are the extensions and how they are linted, it looks like there is some information on the [linter home page](https://docs.deque.com/linter/4.0.0/en/linter-home): 
```
.js, .jsx, and .tsx -> React
.vue -> Vue
.component.html -> Angular
.html and .htm -> HTML
.md and .markdown -> Markdown
```

**Comment by sergei-startsev** [comment]
@JeffreyNordlieTR I think this bug can be considered as closed now
